src/game/mario_step.c

```diff
@@ -404,8 +403,10 @@ struct Surface *check_ledge_grab(struct MarioState *m, struct Surface *prevWall,
     if (ledgeFloor == NULL
         || (*ledgeFloor) == NULL
         || ledgePos[1] < nextPos[1] + 100.0f
+#ifdef DONT_LEDGE_GRAB_STEEP_SLOPES
         || (*ledgeFloor)->normal.y < COS25 // H64 TODO: check if floor is actually slippery
-        || SURFACE_IS_UNSAFE((*ledgeFloor)->type)) {
```

Comment:
disabling qslgs should be a define again soon

Reply:
literally why

who has ever thought "you know what is ruining my hack? qslgs". plus it looks like ass when they're disabled like this, hyena posted a video and it honestly looks far more jank than when they're possible

Reply:
![image](https://user-images.githubusercontent.com/20480418/204370425-1b334cea-52b3-4367-b590-4b311fbce6db.png)
lol

Reply:
still aint gonna be a define get rolled

Reply:
yup some people don't want them and it's not just me, we had a convo about this the other day in jams or something that it sucks to remove this as an option